proxywasm/internal/abi_callback_version.go

```diff
@@ -14,6 +14,13 @@
 
 package internal
 
-//nolint
+import "time"
+
+// nolint
+//
```

Comment:
```suggestion
```

let's delete this as well!

Reply:
Sure - I could only add a "doesn't panic" test for it since there's no logic, hopefully that's OK